docs/source/_static/theme_overrides_SUPERSEDED.css

```diff
@@ -0,0 +1,353 @@
+body {
```

Comment:
@AyanSinhaMahapatra @johnmhoran What is this file? How will it used 
I wonder also if and how the changes in https://github.com/nexB/skeleton/pull/85/files#diff-008dcb3426febd767787b1521f1fe33086313b927ea37eaab86df5fa88a51698R89 affects this? and may mean none of the `_static/` files are used?

Reply:
@pombredanne   If I recall correctly, this file now named `theme_overrides_SUPERSEDED.css` was already in the code when I was asked to widen the pages and deal with mobile -- but it was not being used because of the `conf.py` setting `html_css_files = ["_static/theme_overrides.css"]`.  I fixed that by removing the parent `_static/` reference -- this was already being supplied by current line 76 (formerly 73): `html_static_path = ["_static"]`.  And `theme_overrides_SUPERSEDED.css` should be deleted -- no longer needed.  

@AyanSinhaMahapatra What do you think?  This relates back to changes I made to the vulnerablecode RTD requested by an issue in that repo and a [comment in the PR](https://github.com/nexB/vulnerablecode/pull/1339#pullrequestreview-1734339428) for that issue.

Reply:
@johnmhoran that makes sense, we can get rid of the `theme_overrides_SUPERSEDED.css` file (if the changes are in theme_overrides.css and it works allright), it would be there in history anyway, no need to keep it like this.